filebeat/module/pensando/_meta/config.yml

```diff
@@ -0,0 +1,8 @@
+- module: pensando
+# Firewall logs
+  dfw:
+    enabled: true
+
+    # Set custom paths for the log files. If left empty,
```

Comment:
Would be nice to add all the available config options here even if they are commented.

Reply:
The only config option, currently, is streaming.  Our hardware does not store the logs, nor can we run beats on the hardware yet.

Reply:
Updated and committed: 2980f65fda27cc877fc3026ee909406ca3e8f3bf